### PR TITLE
027 fix dev tests

### PR DESCRIPTION
**Pull request by dITySoftware**
# Make sure these boxes are signed before submitting your Pull Request -- thank you.

# Must haves
- [x] I have read and followed the contributing guide lines at https://github.com/ait-aecid/logdata-anomaly-miner/wiki/Git-development-workflow
- [ ] Issues exist for this PR
- [ ] I added related issues using the "Fixes #123'-notations
- [x] This Pull-Requests merges into the "development"-branch

Fixes #

# Submission specific

- [ ] This PR introduces breaking changes
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

# Describe changes:
-

### CODE REVIEW

File: scripts/testingwrapper.sh [hunk 1]
```diff
@@ -62,6 +67,7 @@ case "$1" in
                 ./runAminerDemo.sh demo/aminer/template_config.py
                 ./runAminerDemo.sh demo/aminer/template_config.yml
                 ./runAminerDemo.sh demo/aminer/demo-config.yml
+                ./runAminerElasticsearchDemo.sh
```
Comment (whotwagner) [inline scripts/testingwrapper.sh:70]: Please to also add this also to the output in the default-case-condition in line 84.

**Review by whotwagner — CHANGES_REQUESTED** [review]

File: scripts/testingwrapper.sh (hunk 1 above)
Comment (dITySoftware) [reply scripts/testingwrapper.sh:70]: Done.

Comment (whotwagner) [reply scripts/testingwrapper.sh:70]: Please rename that file to ./runAminerJsoninputDemo. It's confusing because it's not explicitly for elasticsearch

**Review by whotwagner — CHANGES_REQUESTED** [review]

File: scripts/testingwrapper.sh (hunk 1 above)
Comment (dITySoftware) [reply scripts/testingwrapper.sh]: Done

**Review by whotwagner — APPROVED** [review]

**Pull request merged**